.github/workflows/pull-request.yml

```diff
@@ -1,24 +1,17 @@
-name: 'CI'
+name: "CI"
 on:
   pull_request:
     types: [opened, reopened, synchronize]
-  push:
```

Comment:
Personally I think it's nice to keep this on push because I like having the status in the commit history

Reply:
Makes sense, adding it back now :)